dewy/chunk/router.py

```diff
@@ -20,12 +21,17 @@ async def list_chunks(
     document_id: Annotated[
         int | None, Query(description="Limit to chunks associated with this document")
     ] = None,
-    page: int | None = 1,
+    page: int | None = 0,
```

Comment:
Should probably rev the API in this PR.

Reply:
I'd say lets just rev the library version - I doubt this is in prod anywhere yet and creating versioned API's is sort of a heavy solution.

Reply:
That's what I meant?

Reply:
Cool - I thought you mean adding a `v1` prefix in the paths or something